lib/matplotlib/tests/test_sphinxext.py

```diff
@@ -10,10 +10,31 @@
 import pytest
 
 
-pytest.importorskip('sphinx',
-                    minversion=None if sys.version_info < (3, 10) else '4.1.3')
+def skip_if_no_sphinx(func):
+    pytest.importorskip(
+        'sphinx',
+        minversion=None if sys.version_info < (3, 10) else '4.1.3')
+    return func
 
 
+def _build_sphinx_html(source_dir, doctree_dir, html_dir, extra_args=None):
```

Comment:
@timhoffm  I think you meant to comment here?  I can rename this if you want.

Reply:
Yes, i meant this. Slight preference for renaming, but as said not a big deal either way.